### ISSUE TITLE
Fixed learning rate during pretraining

### ISSUE THREAD

**Issue by mtchiu2** [wxzs5/direct-pretraining#8]
Hi, 

Thank you for your work. I was wondering what is the motivation for using fixed learning rate during pretraining? Is it purely from empirical results or are there any particular reasons?
